CHANGELOG.md

```diff
@@ -7,10 +7,49 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- `cirrus-<stage>-workflow-event` SNS topic, and
+  `WorkflowEventManager` class for managing workflow event actions. ([#261])
+  The actions managed by this class include:
+
+  - updating state of workflows in `StateDB`
+  - announcing interactions cirrus has with a payload to the
+    `cirrus-<stage>-workflow-event`
+    Note: To use this topic, existing deployments will need to add the following
+    to their environment in both their `cirrus.yml` file:
+
+  ```yaml
+  CIRRUS_WORKFLOW_EVENT_TOPIC_ARN: !Ref WorkflowEventTopic
+  ```
+
+  and add the Topic to their `cloudformation/resources.yml` file:
+
+  ```yaml
+  # SNS Topic for any cirrus interactions with a workflow
+  WorkflowEventTopic:
+    Type: "AWS::SNS::Topic"
+    Properties:
+    TopicName: "#{AWS::StackName}-workflow-event"
+  ```
```

Comment:
I think you want this indented the same as a bullet's text so it will be parsed as a continuation of the bullet's content.

```suggestion
    ```yaml
    CIRRUS_WORKFLOW_EVENT_TOPIC_ARN: !Ref WorkflowEventTopic
    ```

    and add the Topic to their `cloudformation/resources.yml` file:

    ```yaml
    # SNS Topic for any cirrus interactions with a workflow
    WorkflowEventTopic:
      Type: "AWS::SNS::Topic"
      Properties:
      TopicName: "#{AWS::StackName}-workflow-event"
    ```
```

Reply:
Applied locally, for rebased reasons.